full/src/test/java/apoc/load/LoadDirectoryTest.java

```diff
@@ -0,0 +1,111 @@
+package apoc.load;
+
+import apoc.util.TestUtil;
+import org.junit.*;
+import org.junit.rules.TemporaryFolder;
+import org.neo4j.configuration.GraphDatabaseSettings;
+import org.neo4j.dbms.api.DatabaseManagementService;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.internal.helpers.collection.Iterators;
+import org.neo4j.test.TestDatabaseManagementServiceBuilder;
+
+import java.io.File;
+import java.util.List;
+import java.util.Map;
+
+import static apoc.ApocConfig.*;
+import static apoc.util.TestUtil.testResult;
+import static org.junit.Assert.*;
+
+
+public class LoadDirectoryTest {
```

Comment:
it would be good to add some tests with `recursive: true`

Reply:
Added 2 test with `{recursive: true}`